### PR TITLE
feat: implement tooltip and detailed view of a POI

### PR DESCRIPTION
**Pull request by lutangar**
Add a detailed, fullscreen, view of all POI information. This view is accessible from a button in the tooltip of a POI.

This roughly implement rules described #249 but should be enough for testing the feature.

This also loosely implements @SeongHunChandra design, but will require some more hard work to get it right.

### CODE REVIEW

**Review by aangelot — APPROVED** [review]
That's a lot of code, don't you want another dev to review it ?

**Comment by lutangar** [comment]
Most code was copied over from __Kepler.gl__  (`LayerHoverInfo`, `MapPopoverFactory`) and isn't up to some standards. 

The codebase would benefits from some more refactoring of these files, but this would it would also bring us farthest from __Kepler.gl__ codebase and make the process of code reconciliation (for updates or feature comparison) more difficult. : this is tradeoff.

This tradeoff isn't obvious to catch and could turn the review process into a pointless exercice. 

BUT given the following disclaimer, some more info on where to find the original __Kelper.gl__ code, and our (and theirs) will to spend some more time training other fellow developers, we could turn this review (and some others to come) into an interesting exercice ! 

Long story short, it means bringing people onboard to make it matters.

**Comment by aangelot** [comment]
So far the only peope we can bring onboard is the "dev factory" team. They can spend a few hours by month i guess, but it would be complicated to ask for more.... 
It looks like we need more if we want relevant reviews ?

**Comment by lutangar** [comment]
> It looks like we need more if we want relevant reviews ?

For features tightly coupled to __Kepler.gl__  yeah, but it's not always the case.
